app/lists.rb

```diff
@@ -0,0 +1,53 @@
+module Listing
+	extend Discordrb::Commands::CommandContainer
+	
+	wing_list = Array.new
+	#The array that is wing_list is created.
+	
+	command :winglist, description: "Lists those who are requesting wing." do |event|
+	#Simple, return the array's contents. Also, admin lock this.
+		break unless event.user.role?(find_role(event, 'wingcaptain'))
```

Comment:
`find_role` won't work as it's not defined in this method. If you're feeling ambitious, you could pull it out into a helper class. Or just pull it into the method how I defined it in admin.rb and add an issue to the bot to do so later. The latter would probably be the better idea.